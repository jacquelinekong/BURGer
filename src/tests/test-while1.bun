int i;
i = 0;
while(i<<5){
	println("pls");
	println("help");
	i = i+1;
}
