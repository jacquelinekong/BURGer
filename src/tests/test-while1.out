pls
help
pls
help
pls
help
pls
help
pls
help
